doc/api/cli.md

```diff
@@ -2018,6 +2018,31 @@ On a machine with 2 GiB of memory, consider setting this to
 $ node --max-old-space-size=1536 index.js
 ```
 
+### `--max_semi_space_size=SIZE` (in megabytes)
```

Comment:
Use dashes (`-`) for consistency with `cli.md`.
> All options, including V8 options, allow words to be separated by both dashes (`-`) or underscores (`_`). For example, `--pending-deprecation` is equivalent to `--pending_deprecation`.
_(src: https://nodejs.org/api/cli.html#options)_

---
```suggestion
### `--max-semi-space-size=SIZE` (in megabytes)
```

Reply:
@peteriman Thanks for your suggestions! I have updated the PR, this is the related [patch](https://github.com/JialuZhang-intel/node/commit/0dfd5e5dbdd748dcd6bfd40c11fed303dfebd4a1).